### PR TITLE
Added Leslie Lamport's TLA+ model-checking tool!

### PR DESCRIPTION
**Pull request by bertmuthalaly**
All the awesome power alluded to in [Thinking for
Programmers](http://channel9.msdn.com/Events/Build/2014/3-642), now
just a `brew cask install tla-plus-toolbox` away!

### CODE REVIEW

File: Casks/tla-plus-toolbox.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+cask :v1 => 'tla-plus-toolbox' do
+  version '1.4.8'
+  sha256 '7f7aec6cc69af1bdbe01e35110939f83b4f880aeec88ce069b9128ab1653da3a'
+
+  url 'https://tla.msr-inria.inria.fr/tlatoolbox/products/TLAToolbox-1.4.8-macosx.cocoa.x86_64.zip'
+  name 'TLA+ Toolbox'
+  homepage 'http://research.microsoft.com/en-us/um/people/lamport/tla/toolbox.html'
+  license :mit
+
+  # for clarity; 'toolbox' is not what you'd spotlight search
```
Comment (vitorgalvao) [inline Casks/tla-plus-toolbox.rb]: Please follow [the standard](https://github.com/caskroom/homebrew-cask/blob/master/doc/CASK_LANGUAGE_REFERENCE.md#target-should-only-be-used-in-select-cases). Use [imagemin.rb](https://github.com/caskroom/homebrew-cask/blob/master/Casks/imagemin.rb) as your reference.

```
# Renamed for clarity: app name is inconsistent with its branding.
# Original discussion: https://github.com/caskroom/homebrew-cask/pull/8829
```

Comment (stijlist) [inline Casks/tla-plus-toolbox.rb]: Sure thing. Do I need to squash the resulting commits or is multiple fine?

On January 11, 2015 at 11:11:24 PM, Vítor Galvão (notifications@github.com) wrote:
In Casks/tla-plus-toolbox.rb:

> @@ -0,0 +1,12 @@
> +cask :v1 => 'tla-plus-toolbox' do
> +  version '1.4.8'
> +  sha256 '7f7aec6cc69af1bdbe01e35110939f83b4f880aeec88ce069b9128ab1653da3a'
> +
> +  url 'https://tla.msr-inria.inria.fr/tlatoolbox/products/TLAToolbox-1.4.8-macosx.cocoa.x86_64.zip'
> +  name 'TLA+ Toolbox'
> +  homepage 'http://research.microsoft.com/en-us/um/people/lamport/tla/toolbox.html'
> +  license :mit
> +
> +  # for clarity; 'toolbox' is not what you'd spotlight search
Please follow the standard. Use imagemin.rb as your reference.

# Renamed for clarity: app name is inconsistent with its branding.
# Original discussion: https://github.com/caskroom/homebrew-cask/pull/8829

**Comment by vitorgalvao** [comment]
This depends on java, so [a caveat](https://github.com/caskroom/homebrew-cask/blob/60fa21bc6582c30be6223050fda62f2e7bacb881/Casks/eid-mw.rb#L13-16) is needed.

File: Casks/tla-plus-toolbox.rb (hunk 1 above)
Comment (vitorgalvao) [inline Casks/tla-plus-toolbox.rb]: Please squash them, yes.

Comment (stijlist) [inline Casks/tla-plus-toolbox.rb]: Updated with caveat, fixed clarity comment, & squashed resulting commits. 

-- 
Bert Muthalaly

On January 11, 2015 at 11:13:48 PM, Vítor Galvão (notifications@github.com) wrote:

In Casks/tla-plus-toolbox.rb:

> @@ -0,0 +1,12 @@
> +cask :v1 => 'tla-plus-toolbox' do
> +  version '1.4.8'
> +  sha256 '7f7aec6cc69af1bdbe01e35110939f83b4f880aeec88ce069b9128ab1653da3a'
> +
> +  url 'https://tla.msr-inria.inria.fr/tlatoolbox/products/TLAToolbox-1.4.8-macosx.cocoa.x86_64.zip'
> +  name 'TLA+ Toolbox'
> +  homepage 'http://research.microsoft.com/en-us/um/people/lamport/tla/toolbox.html'
> +  license :mit
> +
> +  # for clarity; 'toolbox' is not what you'd spotlight search
Please squash them, yes.

File: Casks/tla-plus-toolbox.rb [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  name 'TLA+ Toolbox'
+  homepage 'http://research.microsoft.com/en-us/um/people/lamport/tla/toolbox.html'
+  license :mit
+
+  # Renamed for clarity: app name is inconsistent with its branding.
+  app 'toolbox/toolbox.app', :target => 'TLA+ Toolbox.app'
+
+  caveats <<-EOS.undent
+    #{token} requires Java. You can install the latest version with
+    brew cask install java
```
Comment (vitorgalvao) [inline Casks/tla-plus-toolbox.rb]: This line need to be indented two spaces.

File: Casks/tla-plus-toolbox.rb [hunk 3]
```diff
@@ -0,0 +1,17 @@
+cask :v1 => 'tla-plus-toolbox' do
+  version '1.4.8'
+  sha256 '7f7aec6cc69af1bdbe01e35110939f83b4f880aeec88ce069b9128ab1653da3a'
+
+  url 'https://tla.msr-inria.inria.fr/tlatoolbox/products/TLAToolbox-1.4.8-macosx.cocoa.x86_64.zip'
+  name 'TLA+ Toolbox'
+  homepage 'http://research.microsoft.com/en-us/um/people/lamport/tla/toolbox.html'
+  license :mit
+
+  # Renamed for clarity: app name is inconsistent with its branding.
```
Comment (vitorgalvao) [inline Casks/tla-plus-toolbox.rb]: The other line should also be posted, so people can come back to the issue with ease.

Comment (stijlist) [inline Casks/tla-plus-toolbox.rb]: Done and done. Thanks for your patience! 

-- 
Bert Muthalaly

On January 11, 2015 at 11:22:21 PM, Vítor Galvão (notifications@github.com) wrote:

In Casks/tla-plus-toolbox.rb:

> @@ -0,0 +1,17 @@
> +cask :v1 => 'tla-plus-toolbox' do
> +  version '1.4.8'
> +  sha256 '7f7aec6cc69af1bdbe01e35110939f83b4f880aeec88ce069b9128ab1653da3a'
> +
> +  url 'https://tla.msr-inria.inria.fr/tlatoolbox/products/TLAToolbox-1.4.8-macosx.cocoa.x86_64.zip'
> +  name 'TLA+ Toolbox'
> +  homepage 'http://research.microsoft.com/en-us/um/people/lamport/tla/toolbox.html'
> +  license :mit
> +
> +  # Renamed for clarity: app name is inconsistent with its branding.
The other line should also be posted, so people can come back to the issue with ease.

Comment (vitorgalvao) [inline Casks/tla-plus-toolbox.rb]: Thank you for the submission, and the quick corrections. Merging now.

**Pull request merged**